### PR TITLE
Switch to GitHub Actions.

### PR DESCRIPTION
**Pull request by igorpeshansky**

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+          - 2.6
+          - 2.5
+          - 2.4
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: ruby/setup-ruby@v1
+        env:
+          BUNDLE_GEMFILE: ${{ github.workspace }}/Gemfile
+        with:
```
Comment (qingling128) [inline .github/workflows/ci.yml:18]: Does this automatically handle `gem update bundler` already? If so, feel free to submit. The rest LGTM.

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+          ruby-version: ${{ matrix.rvm }}
+          bundler-cache: true
+          # Run bundle install outside the workspace to keep gems out of the
+          # repo checkout.
+          working-directory: ${{ runner.temp }}
+      - run: |
+          bundle exec rake
+      # TODO: notifications
```
Comment (qingling128) [inline .github/workflows/ci.yml:26]:
```suggestion
      # TODO: send email notifications
```

**Review by qingling128 — APPROVED** [review]
LGTM with a question and a suggested edit.

File: .github/workflows/ci.yml (hunk 1 above)
Comment (igorpeshansky) [reply .github/workflows/ci.yml:18]: It does not update the `bundler` gem explicitly, but I've found that this wasn't needed — the `ruby/setup-ruby` task already installs the latest `bundler`. Using `bundle-cache: true` causes it to run `bundle install` automatically.

File: .github/workflows/ci.yml (hunk 2 above)
Comment (igorpeshansky) [reply .github/workflows/ci.yml]: I've actually embedded the whole Travis notification section in the TODO instead — that way we know exactly what needs to be replicated…

File: .github/workflows/ci.yml (hunk 1 above)
Comment (qingling128) [reply .github/workflows/ci.yml:18]: Got it.

File: .github/workflows/ci.yml (hunk 2 above)
Comment (qingling128) [reply .github/workflows/ci.yml]: Sounds great

File: .github/workflows/ci.yml [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+          BUNDLE_GEMFILE: ${{ github.workspace }}/Gemfile
+        with:
+          ruby-version: ${{ matrix.rvm }}
+          bundler-cache: true
+          # Run bundle install outside the workspace to keep gems out of the
+          # repo checkout.
+          working-directory: ${{ runner.temp }}
+      - run: |
+          bundle exec rake
+      # TODO: notifications:
```
Comment (qingling128) [inline .github/workflows/ci.yml:26]:
```suggestion
      # TODO: notifications similar to the travis settings below:
```

Comment (qingling128) [reply .github/workflows/ci.yml:26]: Clarified that the commented out section are not straight forward GitHub action settings

**Review by qingling128 — APPROVED** [review]
LGTM with a suggestion.

File: .github/workflows/ci.yml (hunk 3 above)
Comment (igorpeshansky) [reply .github/workflows/ci.yml]: The `notifications:` header was actually part of the settings to migrate. Applied in a slightly rephrased way.

**Pull request merged**